Plots/Contours/NCL_polar_1_lg.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May 12 09:38:46 2020
+
+@author: misi1684
+"""
+import numpy as np
+import xarray as xr
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import matplotlib.path as mpath
+import geocat.datafiles as gdf
+import matplotlib.ticker as mticker
+from geocat.viz import util as gvutil
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+U=ds.U[1,:,:]
```

Comment:
General best practice:  Spaces around operators and after commas

(The only time spaces around operators is not good formatting practice is then the operator is in keyword arguments to a function.  For example:

```python
def myfunc(key=value):
    # More code...
```

And when the function is called, like:

```python
myfunc(key=value)
```

And that's it.)

```suggestion
U = ds.U[1, :, :]
```

Reply:
You can use flake8 (https://flake8.pycqa.org/en/latest/) to automatically reformat you code to follow Python formatting conventions.
Just install it and then 
```
flake8 .
```
pointing to the directory you would like to be reformatted.

Reply:
@jukent after running flake8, do I need to recommit/push the python notebook or will changes update automatically here?

Reply:
Running `flake8` changes your files.  So, yes, to get it up to GitHub, you need to recommit and push.